src/test/javascript/spec/app/layouts/search/results.controller.spec.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+describe('Controller Tests', function() {
+
+    describe('Results Controller Tests', function() {
```

Comment:
no functions to test. Removing for now - again, will be required when the endpoint used.